modules/edit-modes/src/lib/measure-angle-mode.js

```diff
@@ -0,0 +1,130 @@
+// @flow
+
+import turfBearing from '@turf/bearing';
+import turfCenter from '@turf/center';
+import memoize from 'memoizee';
```

Comment:
There is a small util file in deck which supports memorize, and avoid introduce another dependency.

https://github.com/uber/deck.gl/blob/master/modules/core/src/utils/memoize.js